orbit/pkg/swiftdialog/run.go

```diff
@@ -52,17 +53,12 @@ const (
 	ExitFileNotFound          ExitCode = 202
 )
 
-func Create(ctx context.Context, swiftDialogBin string, options *SwiftDialogOptions) (*SwiftDialog, error) {
+func Create(ctx context.Context, swiftDialogBin string) (*SwiftDialog, error) {
```

Comment:
I'm curious, why split up the create method into a create and a start?

Reply:
to avoid having a nil pointer in the calling method; this way `Create` returns the pointer immediately and then we can call `Start` async